javaee/api/src/main/java/org/jboss/forge/addon/javaee/jpa/PersistenceOperations.java

```diff
@@ -148,4 +149,11 @@ public JavaResource newEmbeddableEntity(Project project, String entityName, Stri
      * @return the decorated java resource
      */
     public JavaClassSource newEmbeddableEntity(JavaClassSource source);
+
+    /**
+     * Returns the list of all the JPA entities of the project
+     * @param project the current project
+     * @return the list of all the entities of the project
+     */
+    public List<JavaResource> getProjectEntities(Project project);
```

Comment:
Yes, it would, but most of these kind of methods return a List instead of a Set. If you change it to a Set, you end up with plenty of code not working such as the common (entities.indexof) :

```
      Set<JavaResource> entities = persistenceOperations.getProjectEntities(project);
      targetEntity.setValueChoices(entities);
      int idx = -1;
      if (!selection.isEmpty())
      {
         idx = entities.indexOf(selection.get());
      }
      if (idx != -1)
      {
         targetEntity.setDefaultValue(entities.get(idx));
      }
      builder.add(named).add(query).add(targetEntity);
```